python/huggingfaceserver/health_check.py

```diff
@@ -13,41 +13,88 @@
 # limitations under the License.
 
 import argparse
+import sys
+import time
+import os
+
 import ray
 import requests
-import sys
 from kserve.logging import logger
 
 
-def initialize_ray_cluster():
-    if not ray.is_initialized():  # Check if Ray is already initialized
-        ray.init(address="auto")
-        return "Ray initialized"
-    else:
-        return "Ray already initialized"
+def initialize_ray_cluster(ray_address="auto"):
+    try:
+        if ray.is_initialized():  # Check if Ray is already initialized
+            return
+        else:
+            ray.init(address=ray_address)
```

Comment:
Is it a good idea ti initialize the ray cluster in a health_check script?
imho it should be somewhere else.

Reply:
initialize is for connection to the head node. it is not starting a new ray cluster

Reply:
yeah, just realize that ray.init with no parameters would try to start a new one :)
Thanks.